tasks/codex/edm.py

```diff
@@ -2,12 +2,16 @@
 """
 Data transfer from EDM to houston Invoke.
 """
+import logging
+
 import tqdm
 from flask import current_app as app
 
 from flask_restx_patched import is_extension_enabled
 from tasks.utils import app_context_task
 
+log = logging.getLogger(__name__)  # pylint: disable=invalid-name
```

Comment:
Why? 
Logging is surely for what we want to go into the system logs but what benefit is that here?
I'd actually say tha t we would be better off ensuring that this transferred data has appropriate audit logs as that will aid tracability

Reply:
i originally did this cuz i saw some tasks using logs, while others used just `print()` to convey what was happening.
audit may not be a bad idea but not sure we want it for _every_ object?  or maybe we dont care.  i guess its one-time only.  i will think about it as i keep testing/running these on real data.

Reply:
I would say that   we do want audit logs for every object as otherwise we'd have no autit record of how they were created. 
I think it's now time to revisit this to make sure that the logging is what we need.

Reply:
just for the record, nothing is being _created_ here, per se.  it is _setting fields_ on existing data.  and: it will set the field on almost every item (it iterates thru sighting, individual, and encounter `query.all()`).   just not sure how much data auditing each one would give us, see what i mean?  its just going to say "we added stuff to guid X" for every X.  and unless we this audit records _which fields_ we changed, i dont think we gain any more info than adding a _single_ log "we altered all encounters".  (and at that point we probably _double_ the time it takes since we are writing to the db 2x as much.

granted this is all a **one-time process** so i am not sure how much it actually matters (any of it!) but i just was trying to keep the inside of that loop as minimal as possible, esp since i started at 12 hours (for a small codex!).   :grimacing: 

thoughts?

Reply:
Well we do auditlog for each time an existing thing is patched which includes what fields changed to what values and by who. 
I can see the point on the timing though. You're the one with the more experience of debugging so I'll let you make the call on how usefl this actually is.

Reply:
i am going to make the call to _not_ log here.  if for no other reason than basically **if** we have a migrated old-world wildbook we _know_ we will have run this step. and it is awful to run (needs the edm etc) and thus hard to do accidentally.  so basically its implied this step had to happen.  :grin:   any new/fresh/empty codex will never ever have any thing to do with this (thank goodness).  or better not.  haha